test/Client/AnnualReportsTest.php

```diff
@@ -127,6 +127,71 @@ public function it_gets_an_annual_report()
 
     /**
      * @test
+     */
```

Comment:
Given the Client consistently use `ArrayFromIterator`, it doesn't scale very much to test all of them for the same behavior. If we have a `ArrayFromIteratorTest` such as:
```
class ArrayFromIteratorExample
{
    use ArrayFromIterator;
    // some code to make $this iterable
}

class ArrayFromIteratorTest extends \PHPUnit_Framework_TestCase
{
    // test what a ArrayFromIteratorExample instance does
}
```
PHPUnit provides `getMockForTrait` https://phpunit.de/manual/current/en/test-doubles.html for creating the example object, but it's not very readable and for this use case having a concrete real class documents both makes reusing it easier and documents how to use the Trait. It could be an anonymous class though.